docs/resources/redis_cluster.md

```diff
@@ -153,13 +139,34 @@ The `acl` block supports:
 
 The `private_network` block supports :
 
-- `id` - (Required) The UUID of the private network resource.
-- `service_ips` - (Required) Endpoint IPv4 addresses
-  in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation). You must provide at
-  least one IP per node or The IP network address within the private subnet is determined by the IP Address Management (IPAM)
-  service if not set.
+- `id` - (Required) The UUID of the Private Network resource.
+- `service_ips` - (Required) Endpoint IPv4 addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation). You must provide at least one IP per node.
```

Comment:
`services_ips` are documented as required. Should it be optional as we can omit them to use IPAM ?